src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * 
+ * Represents a Person's address in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ *
+ */
+
+public class Contact {
+	
+	public final String contactInfo;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.